www/experiments/custom.inc

```diff
@@ -68,6 +72,26 @@ $assessment[$category]["opportunities"][] = array(
     "inputttext" => true
 );
 
+if( $requests[0]['body_url'] ){
+    $assessment[$category]["opportunities"][] = array(
+        "title" =>  "Edit Response HTML",
+        "desc" =>  "This experiment allows you to freely edit the text of the initial HTML response",
+        "examples" =>  array(),
+        "experiments" =>  array(
+            (object) [
+                'id' => '054',
+                'title' => 'Edit Response HTML',
+                "desc" => '<p>This experiment will replace the initial HTML with the submitted value as if it was served that way initially.</p>',
+                "expvar" => 'editresponsehtml',
+                'textinputvalue' => htmlentities(file_get_contents("https://www.webpagetest.org" . $requests[0]['body_url'])),
```

Comment:
hmm, isn't there a better way to fetch the body? We can address this todo maybe: 
https://github.com/WPO-Foundation/webpagetest/blob/2fabc42e7a87af394f363098ac01ad89bd9f4fd7/www/htmldiff.php#L39

Reply:
Would love to see that moved to a utility sure.